astropy/coordinates/tests/test_frames.py

```diff
@@ -1044,3 +1044,25 @@ class TestFrame(BaseCoordinateFrame):
         attrtest = Attribute(default=Spam())
 
     assert "TEST REPR" in repr(TestFrame())
+
+
+def test_component_names_repr():
+    from astropy.coordinates.baseframe import BaseCoordinateFrame, RepresentationMapping
```

Comment:
why are these imports are here, rather than at the top?

Reply:
I just followed the structure of the stuff that was there, but did also wonder about it. Seems good for a clean-up PR...

Reply:
OK, cool. I plan to do more pep8 stuff cleanup before the release, so we can leave it for that.